config/initializers/s3.rb

```diff
@@ -0,0 +1,9 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider: 'AWS',
+    aws_access_key_id: ENV['S3_KEY'],
+    aws_secret_access_key: ENV['S3_SECRET'],
+    region: ENV['S3_REGION']
+  }
+  config.fog_directory  = ENV['S3_BUCKET']
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.